internal/api/grpc/server/middleware/execution_interceptor.go

```diff
@@ -89,18 +89,27 @@ type ContextInfoRequest struct {
 }
 
 func (c *ContextInfoRequest) GetHTTPRequestBody() []byte {
-	data, err := json.Marshal(c)
+	jsonpb := &runtime.JSONPb{
+		UnmarshalOptions: protojson.UnmarshalOptions{
+			DiscardUnknown: true,
+			AllowPartial:   true,
+		},
+	}
+	data, err := jsonpb.Marshal(c)
```

Comment:
marshalling a non proto message as root object will have the same effect as using json package